From 06bcab4ef99d3c57dcd7f599fdb43f1e402011bf Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Tue, 4 Mar 2025 18:15:19 +0100
Subject: [PATCH 1621/1695] dt-bindings: stm32: remove CCB bindings

CCB peripheral is no more supported on STM32MP21.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f369d3815f14942c7c9d6fbefb9a11cf784763a7 dt-bindings: stm32: remove CCB bindings]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: Ie638d0f4b8446b759008e88e8f17bdcde08a6b4f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/441747
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/include/dt-bindings/clock/st,stm32mp21-rcc.h | 1 -
 core/include/dt-bindings/reset/st,stm32mp21-rcc.h | 1 -
 core/include/dt-bindings/soc/stm32mp21-rifsc.h    | 1 -
 3 files changed, 3 deletions(-)

diff --git a/core/include/dt-bindings/clock/st,stm32mp21-rcc.h b/core/include/dt-bindings/clock/st,stm32mp21-rcc.h
index 4f4dccdf1..30d90c6c2 100644
--- a/core/include/dt-bindings/clock/st,stm32mp21-rcc.h
+++ b/core/include/dt-bindings/clock/st,stm32mp21-rcc.h
@@ -143,7 +143,6 @@
 #define CK_BUS_GPIOH		120
 #define CK_BUS_GPIOI		121
 #define CK_BUS_GPIOZ		122
-#define CK_BUS_CCB		123
 #define CK_BUS_RTC		124
 #define CK_BUS_LPUART1		125
 #define CK_BUS_LPTIM3		126
diff --git a/core/include/dt-bindings/reset/st,stm32mp21-rcc.h b/core/include/dt-bindings/reset/st,stm32mp21-rcc.h
index bb9d3cef0..b412a8bfa 100644
--- a/core/include/dt-bindings/reset/st,stm32mp21-rcc.h
+++ b/core/include/dt-bindings/reset/st,stm32mp21-rcc.h
@@ -122,7 +122,6 @@
 #define DDRCFG_R	112
 #define DDR_R		113
 #define DDRPERFM_R	114
-#define CCB_R		115
 #define IWDG1_SYS_R	116
 #define IWDG2_SYS_R	117
 #define IWDG3_SYS_R	118
diff --git a/core/include/dt-bindings/soc/stm32mp21-rifsc.h b/core/include/dt-bindings/soc/stm32mp21-rifsc.h
index 6b896766f..6aba0c56f 100644
--- a/core/include/dt-bindings/soc/stm32mp21-rifsc.h
+++ b/core/include/dt-bindings/soc/stm32mp21-rifsc.h
@@ -68,7 +68,6 @@
 #define STM32MP21_RIFSC_CSI_ID			86
 #define STM32MP21_RIFSC_DCMIPP_ID		87
 #define STM32MP21_RIFSC_DCMI_PSSI_ID		88
-#define STM32MP21_RIFSC_CCB_ID			91
 #define STM32MP21_RIFSC_RNG1_ID			92
 #define STM32MP21_RIFSC_RNG2_ID			93
 #define STM32MP21_RIFSC_PKA_ID			94
-- 
2.47.2

